.jenkins/scripts/build_keycloak_opa-s390x.sh

```diff
@@ -2,8 +2,8 @@
 
 eval $(minikube docker-env)
 
-sed -i 's#kubectl apply -n \${NAMESPACE} -f https://github.com/keycloak/keycloak-operator/raw/\${KEYCLOAK_VERSION}/deploy/operator.yaml#curl -s https://raw.githubusercontent.com/keycloak/keycloak-operator/\${KEYCLOAK_VERSION}/deploy/operator.yaml | sed "s\#Always\#IfNotPresent\#g" | kubectl apply -n \${NAMESPACE} -f -#g' systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh
-sed -i 's#kubectl delete -n \${NAMESPACE} -f https://github.com/keycloak/keycloak-operator/raw/\${KEYCLOAK_VERSION}/deploy/operator.yaml#curl -s https://raw.githubusercontent.com/keycloak/keycloak-operator/\${KEYCLOAK_VERSION}/deploy/operator.yaml | sed "s\#Always\#IfNotPresent\#g" | kubectl delete -n \${NAMESPACE} -f -#g' systemtest/src/test/resources/oauth2/teardown_keycloak_operator.sh
+sed -i 's#kubectl apply -n \${KEYCLOAK_OPERATOR_NAMESPACE} -f https://github.com/keycloak/keycloak-operator/raw/\${KEYCLOAK_VERSION}/deploy/operator.yaml#curl -s https://raw.githubusercontent.com/keycloak/keycloak-operator/\${KEYCLOAK_VERSION}/deploy/operator.yaml | sed "s\#Always\#IfNotPresent\#g" | kubectl apply -n \${KEYCLOAK_OPERATOR_NAMESPACE} -f -#g' systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh
```

Comment:
From where does it load `KEYCLOAK_OPERATOR_NAMESPACE` ?

Reply:
Thanks @Frawless. The variable `KEYCLOAK_OPERATOR_NAMESPACE` is loaded in `prepare_keycloak_operator.sh` and `teardown_keycloak_operator.sh` under `systemtest/src/test/resources/oauth2/`, not in this patch.

This patch is just to replace the value `Always` with `IfNotPresent` for ImagePullPolicy in `operator.yaml` before running `kubectl apply` or `kubectl delete`, so that `docker` will not pull `keycloak-operator` image from quay.io which does not have s390x support yet, but launch the locally built `keycloak-operator` image instead.

Reply:
Sorry, I had old revision of the code.